[2024-07-08 20:25:56,623] INFO [GroupCoordinator 0]: Member kafka-python-2.0.2-2f2d568b-3b07-4bab-b14b-558acd147f66 in group mygroup has failed, removing it from the group (kafka.coordinator.group.GroupCoordinator)
[2024-07-08 20:25:56,634] INFO Expiring session 0x1003acc34310003, timeout of 18000ms exceeded (org.apache.zookeeper.server.ZooKeeperServer)
[2024-07-08 20:25:56,841] WARN Attempting to send response via channel for which there is no open connection, connection id 172.27.48.1:9092-172.27.48.1:61636-3 (kafka.network.Processor)
[2024-07-08 20:25:57,187] INFO [GroupCoordinator 0]: Preparing to rebalance group mygroup in state PreparingRebalance with old generation 24 (__consumer_offsets-39) (reason: removing member kafka-python-2.0.2-2f2d568b-3b07-4bab-b14b-558acd147f66 on heartbeat expiration) (kafka.coordinator.group.GroupCoordinator)
[2024-07-08 20:25:57,308] WARN Client session timed out, have not heard from server in 30705987ms for session id 0x1003acc34310003 (org.apache.zookeeper.ClientCnxn)
[2024-07-08 20:25:57,350] WARN Session 0x1003acc34310003 for server localhost/[0:0:0:0:0:0:0:1]:2181, Closing socket connection. Attempting reconnect except it is a SessionExpiredException. (org.apache.zookeeper.ClientCnxn)
org.apache.zookeeper.ClientCnxn$SessionTimeoutException: Client session timed out, have not heard from server in 30705987ms for session id 0x1003acc34310003
	at org.apache.zookeeper.ClientCnxn$SendThread.run(ClientCnxn.java:1257)
[2024-07-08 20:25:57,348] INFO [GroupCoordinator 0]: Group mygroup with generation 25 is now empty (__consumer_offsets-39) (kafka.coordinator.group.GroupCoordinator)
[2024-07-08 20:25:57,360] INFO [GroupCoordinator 0]: Dynamic Member with unknown member id joins group mygroup in Empty state. Created a new member id kafka-python-2.0.2-20587188-3e2c-4dfe-a582-55316cfe0905 for this member and add to the group. (kafka.coordinator.group.GroupCoordinator)
[2024-07-08 20:25:57,410] INFO [GroupCoordinator 0]: Preparing to rebalance group mygroup in state PreparingRebalance with old generation 25 (__consumer_offsets-39) (reason: Adding new member kafka-python-2.0.2-20587188-3e2c-4dfe-a582-55316cfe0905 with group instance id None; client reason: not provided) (kafka.coordinator.group.GroupCoordinator)
[2024-07-08 20:25:57,435] INFO [GroupCoordinator 0]: Stabilized group mygroup generation 26 (__consumer_offsets-39) with 1 members (kafka.coordinator.group.GroupCoordinator)
[2024-07-08 20:25:57,462] INFO [GroupCoordinator 0]: Assignment received from leader kafka-python-2.0.2-20587188-3e2c-4dfe-a582-55316cfe0905 for group mygroup for generation 26. The group has 1 members, 0 of which are static. (kafka.coordinator.group.GroupCoordinator)
[2024-07-08 20:25:59,480] INFO Opening socket connection to server localhost/127.0.0.1:2181. (org.apache.zookeeper.ClientCnxn)
[2024-07-08 20:25:59,482] INFO Socket connection established, initiating session, client: /127.0.0.1:61645, server: localhost/127.0.0.1:2181 (org.apache.zookeeper.ClientCnxn)
[2024-07-08 20:25:59,486] INFO Invalid session 0x1003acc34310003 for client /127.0.0.1:61645, probably expired (org.apache.zookeeper.server.ZooKeeperServer)
[2024-07-08 20:25:59,487] WARN Unable to reconnect to ZooKeeper service, session 0x1003acc34310003 has expired (org.apache.zookeeper.ClientCnxn)
[2024-07-08 20:25:59,487] INFO EventThread shut down for session: 0x1003acc34310003 (org.apache.zookeeper.ClientCnxn)
[2024-07-08 20:25:59,487] WARN Session 0x1003acc34310003 for server localhost/127.0.0.1:2181, Closing socket connection. Attempting reconnect except it is a SessionExpiredException. (org.apache.zookeeper.ClientCnxn)
org.apache.zookeeper.ClientCnxn$SessionExpiredException: Unable to reconnect to ZooKeeper service, session 0x1003acc34310003 has expired
	at org.apache.zookeeper.ClientCnxn$SendThread.onConnected(ClientCnxn.java:1439)
	at org.apache.zookeeper.ClientCnxnSocket.readConnectResult(ClientCnxnSocket.java:154)
	at org.apache.zookeeper.ClientCnxnSocketNIO.doIO(ClientCnxnSocketNIO.java:86)
	at org.apache.zookeeper.ClientCnxnSocketNIO.doTransport(ClientCnxnSocketNIO.java:350)
	at org.apache.zookeeper.ClientCnxn$SendThread.run(ClientCnxn.java:1289)
[2024-07-08 20:25:59,488] INFO [ZooKeeperClient Kafka server] Session expired. (kafka.zookeeper.ZooKeeperClient)
[2024-07-08 20:25:59,490] INFO [ZooKeeperClient Kafka server] Initializing a new session to localhost:2181. (kafka.zookeeper.ZooKeeperClient)
[2024-07-08 20:25:59,492] INFO Initiating client connection, connectString=localhost:2181 sessionTimeout=18000 watcher=kafka.zookeeper.ZooKeeperClient$ZooKeeperClientWatcher$@147a5d08 (org.apache.zookeeper.ZooKeeper)
[2024-07-08 20:25:59,501] INFO jute.maxbuffer value is 4194304 Bytes (org.apache.zookeeper.ClientCnxnSocket)
[2024-07-08 20:25:59,503] INFO zookeeper.request.timeout value is 0. feature enabled=false (org.apache.zookeeper.ClientCnxn)
[2024-07-08 20:25:59,505] INFO Opening socket connection to server localhost/[0:0:0:0:0:0:0:1]:2181. (org.apache.zookeeper.ClientCnxn)
[2024-07-08 20:25:59,505] INFO Creating /brokers/ids/0 (is it secure? false) (kafka.zk.KafkaZkClient)
[2024-07-08 20:25:59,507] INFO Socket connection established, initiating session, client: /[0:0:0:0:0:0:0:1]:61646, server: localhost/[0:0:0:0:0:0:0:1]:2181 (org.apache.zookeeper.ClientCnxn)
[2024-07-08 20:25:59,514] INFO Session establishment complete on server localhost/[0:0:0:0:0:0:0:1]:2181, session id = 0x1003acc34310004, negotiated timeout = 18000 (org.apache.zookeeper.ClientCnxn)
[2024-07-08 20:25:59,516] INFO [MetadataCache brokerId=0] Updated cache from existing Some(Features(version=3.7-IV4, finalizedFeatures={}, finalizedFeaturesEpoch=0)) to latest Features(version=3.7-IV4, finalizedFeatures={}, finalizedFeaturesEpoch=0). (kafka.server.metadata.ZkMetadataCache)
[2024-07-08 20:25:59,520] INFO Stat of the created znode at /brokers/ids/0 is: 257,257,1720450559515,1720450559515,1,0,0,72122242761293828,196,0,257
 (kafka.zk.KafkaZkClient)
[2024-07-08 20:25:59,521] INFO Registered broker 0 at path /brokers/ids/0 with addresses: PLAINTEXT://anshul:9092, czxid (broker epoch): 257 (kafka.zk.KafkaZkClient)
[2024-07-08 21:42:43,390] INFO [GroupCoordinator 0]: Member kafka-python-2.0.2-20587188-3e2c-4dfe-a582-55316cfe0905 in group mygroup has failed, removing it from the group (kafka.coordinator.group.GroupCoordinator)
[2024-07-08 21:42:43,394] INFO [GroupCoordinator 0]: Preparing to rebalance group mygroup in state PreparingRebalance with old generation 26 (__consumer_offsets-39) (reason: removing member kafka-python-2.0.2-20587188-3e2c-4dfe-a582-55316cfe0905 on heartbeat expiration) (kafka.coordinator.group.GroupCoordinator)
[2024-07-08 21:42:43,395] INFO [GroupCoordinator 0]: Group mygroup with generation 27 is now empty (__consumer_offsets-39) (kafka.coordinator.group.GroupCoordinator)
[2024-07-08 21:42:43,408] WARN Attempting to send response via channel for which there is no open connection, connection id fe80:0:0:0:447d:591f:badf:5ff5%57:9092-fe80:0:0:0:447d:591f:badf:5ff5%57:61642-4 (kafka.network.Processor)
[2024-07-08 21:42:43,627] INFO [GroupCoordinator 0]: Dynamic Member with unknown member id joins group mygroup in Empty state. Created a new member id kafka-python-2.0.2-823d7594-6605-4db6-acce-6d63124f9121 for this member and add to the group. (kafka.coordinator.group.GroupCoordinator)
[2024-07-08 21:42:43,627] INFO [GroupCoordinator 0]: Preparing to rebalance group mygroup in state PreparingRebalance with old generation 27 (__consumer_offsets-39) (reason: Adding new member kafka-python-2.0.2-823d7594-6605-4db6-acce-6d63124f9121 with group instance id None; client reason: not provided) (kafka.coordinator.group.GroupCoordinator)
[2024-07-08 21:42:43,629] INFO [GroupCoordinator 0]: Stabilized group mygroup generation 28 (__consumer_offsets-39) with 1 members (kafka.coordinator.group.GroupCoordinator)
[2024-07-08 21:42:43,634] INFO [GroupCoordinator 0]: Assignment received from leader kafka-python-2.0.2-823d7594-6605-4db6-acce-6d63124f9121 for group mygroup for generation 28. The group has 1 members, 0 of which are static. (kafka.coordinator.group.GroupCoordinator)
[2024-07-08 21:42:45,987] INFO Expiring session 0x1003acc34310004, timeout of 18000ms exceeded (org.apache.zookeeper.server.ZooKeeperServer)
[2024-07-08 21:42:46,092] WARN Session 0x1003acc34310004 for server localhost/[0:0:0:0:0:0:0:1]:2181, Closing socket connection. Attempting reconnect except it is a SessionExpiredException. (org.apache.zookeeper.ClientCnxn)
EndOfStreamException: Unable to read additional data from server sessionid 0x1003acc34310004, likely server has closed socket
	at org.apache.zookeeper.ClientCnxnSocketNIO.doIO(ClientCnxnSocketNIO.java:77)
	at org.apache.zookeeper.ClientCnxnSocketNIO.doTransport(ClientCnxnSocketNIO.java:350)
	at org.apache.zookeeper.ClientCnxn$SendThread.run(ClientCnxn.java:1289)
[2024-07-08 21:42:47,731] INFO Opening socket connection to server localhost/127.0.0.1:2181. (org.apache.zookeeper.ClientCnxn)
[2024-07-08 21:42:47,735] INFO Socket connection established, initiating session, client: /127.0.0.1:62560, server: localhost/127.0.0.1:2181 (org.apache.zookeeper.ClientCnxn)
[2024-07-08 21:42:47,747] INFO Invalid session 0x1003acc34310004 for client /127.0.0.1:62560, probably expired (org.apache.zookeeper.server.ZooKeeperServer)
[2024-07-08 21:42:47,749] INFO EventThread shut down for session: 0x1003acc34310004 (org.apache.zookeeper.ClientCnxn)
[2024-07-08 21:42:47,749] INFO [ZooKeeperClient Kafka server] Session expired. (kafka.zookeeper.ZooKeeperClient)
[2024-07-08 21:42:47,749] WARN Unable to reconnect to ZooKeeper service, session 0x1003acc34310004 has expired (org.apache.zookeeper.ClientCnxn)
[2024-07-08 21:42:47,751] WARN Session 0x1003acc34310004 for server localhost/127.0.0.1:2181, Closing socket connection. Attempting reconnect except it is a SessionExpiredException. (org.apache.zookeeper.ClientCnxn)
org.apache.zookeeper.ClientCnxn$SessionExpiredException: Unable to reconnect to ZooKeeper service, session 0x1003acc34310004 has expired
	at org.apache.zookeeper.ClientCnxn$SendThread.onConnected(ClientCnxn.java:1439)
	at org.apache.zookeeper.ClientCnxnSocket.readConnectResult(ClientCnxnSocket.java:154)
	at org.apache.zookeeper.ClientCnxnSocketNIO.doIO(ClientCnxnSocketNIO.java:86)
	at org.apache.zookeeper.ClientCnxnSocketNIO.doTransport(ClientCnxnSocketNIO.java:350)
	at org.apache.zookeeper.ClientCnxn$SendThread.run(ClientCnxn.java:1289)
[2024-07-08 21:42:47,754] INFO [ZooKeeperClient Kafka server] Initializing a new session to localhost:2181. (kafka.zookeeper.ZooKeeperClient)
[2024-07-08 21:42:47,755] INFO Initiating client connection, connectString=localhost:2181 sessionTimeout=18000 watcher=kafka.zookeeper.ZooKeeperClient$ZooKeeperClientWatcher$@147a5d08 (org.apache.zookeeper.ZooKeeper)
[2024-07-08 21:42:47,777] INFO jute.maxbuffer value is 4194304 Bytes (org.apache.zookeeper.ClientCnxnSocket)
[2024-07-08 21:42:47,781] INFO zookeeper.request.timeout value is 0. feature enabled=false (org.apache.zookeeper.ClientCnxn)
[2024-07-08 21:42:47,870] INFO Creating /brokers/ids/0 (is it secure? false) (kafka.zk.KafkaZkClient)
[2024-07-08 21:42:47,946] INFO Opening socket connection to server localhost/127.0.0.1:2181. (org.apache.zookeeper.ClientCnxn)
[2024-07-08 21:42:48,021] INFO Socket connection established, initiating session, client: /127.0.0.1:62561, server: localhost/127.0.0.1:2181 (org.apache.zookeeper.ClientCnxn)
[2024-07-08 21:42:48,066] INFO Session establishment complete on server localhost/127.0.0.1:2181, session id = 0x1003acc34310005, negotiated timeout = 18000 (org.apache.zookeeper.ClientCnxn)
[2024-07-08 21:42:48,093] INFO Stat of the created znode at /brokers/ids/0 is: 262,262,1720455168072,1720455168072,1,0,0,72122242761293829,196,0,262
 (kafka.zk.KafkaZkClient)
[2024-07-08 21:42:48,111] INFO Registered broker 0 at path /brokers/ids/0 with addresses: PLAINTEXT://anshul:9092, czxid (broker epoch): 262 (kafka.zk.KafkaZkClient)
[2024-07-08 21:42:48,122] INFO [MetadataCache brokerId=0] Updated cache from existing Some(Features(version=3.7-IV4, finalizedFeatures={}, finalizedFeaturesEpoch=0)) to latest Features(version=3.7-IV4, finalizedFeatures={}, finalizedFeaturesEpoch=0). (kafka.server.metadata.ZkMetadataCache)
[2024-07-09 05:31:20,644] WARN Attempting to send response via channel for which there is no open connection, connection id 172.27.48.1:9092-172.27.48.1:62540-5 (kafka.network.Processor)
[2024-07-09 05:31:20,805] INFO [GroupCoordinator 0]: Member kafka-python-2.0.2-823d7594-6605-4db6-acce-6d63124f9121 in group mygroup has failed, removing it from the group (kafka.coordinator.group.GroupCoordinator)
[2024-07-09 05:31:20,944] INFO [GroupCoordinator 0]: Preparing to rebalance group mygroup in state PreparingRebalance with old generation 28 (__consumer_offsets-39) (reason: removing member kafka-python-2.0.2-823d7594-6605-4db6-acce-6d63124f9121 on heartbeat expiration) (kafka.coordinator.group.GroupCoordinator)
[2024-07-09 05:31:20,945] INFO [GroupCoordinator 0]: Group mygroup with generation 29 is now empty (__consumer_offsets-39) (kafka.coordinator.group.GroupCoordinator)
[2024-07-09 05:31:20,975] INFO [GroupCoordinator 0]: Dynamic Member with unknown member id joins group mygroup in Empty state. Created a new member id kafka-python-2.0.2-29ffda6e-4354-4d04-9805-bd6dca27030f for this member and add to the group. (kafka.coordinator.group.GroupCoordinator)
[2024-07-09 05:31:20,976] INFO [GroupCoordinator 0]: Preparing to rebalance group mygroup in state PreparingRebalance with old generation 29 (__consumer_offsets-39) (reason: Adding new member kafka-python-2.0.2-29ffda6e-4354-4d04-9805-bd6dca27030f with group instance id None; client reason: not provided) (kafka.coordinator.group.GroupCoordinator)
[2024-07-09 05:31:20,977] INFO [GroupCoordinator 0]: Stabilized group mygroup generation 30 (__consumer_offsets-39) with 1 members (kafka.coordinator.group.GroupCoordinator)
[2024-07-09 05:31:20,980] INFO [GroupCoordinator 0]: Assignment received from leader kafka-python-2.0.2-29ffda6e-4354-4d04-9805-bd6dca27030f for group mygroup for generation 30. The group has 1 members, 0 of which are static. (kafka.coordinator.group.GroupCoordinator)
[2024-07-09 05:31:22,352] INFO Expiring session 0x1003acc34310005, timeout of 18000ms exceeded (org.apache.zookeeper.server.ZooKeeperServer)
[2024-07-09 05:31:22,375] WARN Client session timed out, have not heard from server in 26383797ms for session id 0x1003acc34310005 (org.apache.zookeeper.ClientCnxn)
[2024-07-09 05:31:22,376] WARN Session 0x1003acc34310005 for server localhost/127.0.0.1:2181, Closing socket connection. Attempting reconnect except it is a SessionExpiredException. (org.apache.zookeeper.ClientCnxn)
org.apache.zookeeper.ClientCnxn$SessionTimeoutException: Client session timed out, have not heard from server in 26383797ms for session id 0x1003acc34310005
	at org.apache.zookeeper.ClientCnxn$SendThread.run(ClientCnxn.java:1257)
[2024-07-09 05:31:23,844] INFO Opening socket connection to server localhost/[0:0:0:0:0:0:0:1]:2181. (org.apache.zookeeper.ClientCnxn)
[2024-07-09 05:31:23,846] INFO Socket connection established, initiating session, client: /[0:0:0:0:0:0:0:1]:62885, server: localhost/[0:0:0:0:0:0:0:1]:2181 (org.apache.zookeeper.ClientCnxn)
[2024-07-09 05:31:23,850] INFO Invalid session 0x1003acc34310005 for client /[0:0:0:0:0:0:0:1]:62885, probably expired (org.apache.zookeeper.server.ZooKeeperServer)
[2024-07-09 05:31:23,853] WARN Unable to reconnect to ZooKeeper service, session 0x1003acc34310005 has expired (org.apache.zookeeper.ClientCnxn)
[2024-07-09 05:31:23,853] INFO [ZooKeeperClient Kafka server] Session expired. (kafka.zookeeper.ZooKeeperClient)
[2024-07-09 05:31:23,853] INFO EventThread shut down for session: 0x1003acc34310005 (org.apache.zookeeper.ClientCnxn)
[2024-07-09 05:31:23,854] WARN Session 0x1003acc34310005 for server localhost/[0:0:0:0:0:0:0:1]:2181, Closing socket connection. Attempting reconnect except it is a SessionExpiredException. (org.apache.zookeeper.ClientCnxn)
org.apache.zookeeper.ClientCnxn$SessionExpiredException: Unable to reconnect to ZooKeeper service, session 0x1003acc34310005 has expired
	at org.apache.zookeeper.ClientCnxn$SendThread.onConnected(ClientCnxn.java:1439)
	at org.apache.zookeeper.ClientCnxnSocket.readConnectResult(ClientCnxnSocket.java:154)
	at org.apache.zookeeper.ClientCnxnSocketNIO.doIO(ClientCnxnSocketNIO.java:86)
	at org.apache.zookeeper.ClientCnxnSocketNIO.doTransport(ClientCnxnSocketNIO.java:350)
	at org.apache.zookeeper.ClientCnxn$SendThread.run(ClientCnxn.java:1289)
[2024-07-09 05:31:23,854] INFO [ZooKeeperClient Kafka server] Initializing a new session to localhost:2181. (kafka.zookeeper.ZooKeeperClient)
[2024-07-09 05:31:23,861] INFO Initiating client connection, connectString=localhost:2181 sessionTimeout=18000 watcher=kafka.zookeeper.ZooKeeperClient$ZooKeeperClientWatcher$@147a5d08 (org.apache.zookeeper.ZooKeeper)
[2024-07-09 05:31:23,866] INFO jute.maxbuffer value is 4194304 Bytes (org.apache.zookeeper.ClientCnxnSocket)
[2024-07-09 05:31:23,868] INFO zookeeper.request.timeout value is 0. feature enabled=false (org.apache.zookeeper.ClientCnxn)
[2024-07-09 05:31:23,870] INFO Opening socket connection to server localhost/[0:0:0:0:0:0:0:1]:2181. (org.apache.zookeeper.ClientCnxn)
[2024-07-09 05:31:23,871] INFO Socket connection established, initiating session, client: /[0:0:0:0:0:0:0:1]:62886, server: localhost/[0:0:0:0:0:0:0:1]:2181 (org.apache.zookeeper.ClientCnxn)
[2024-07-09 05:31:23,875] INFO Creating /brokers/ids/0 (is it secure? false) (kafka.zk.KafkaZkClient)
[2024-07-09 05:31:23,878] INFO Session establishment complete on server localhost/[0:0:0:0:0:0:0:1]:2181, session id = 0x1003acc34310006, negotiated timeout = 18000 (org.apache.zookeeper.ClientCnxn)
[2024-07-09 05:31:23,884] INFO [MetadataCache brokerId=0] Updated cache from existing Some(Features(version=3.7-IV4, finalizedFeatures={}, finalizedFeaturesEpoch=0)) to latest Features(version=3.7-IV4, finalizedFeatures={}, finalizedFeaturesEpoch=0). (kafka.server.metadata.ZkMetadataCache)
[2024-07-09 05:31:23,892] INFO Stat of the created znode at /brokers/ids/0 is: 267,267,1720483283882,1720483283882,1,0,0,72122242761293830,196,0,267
 (kafka.zk.KafkaZkClient)
[2024-07-09 05:31:23,893] INFO Registered broker 0 at path /brokers/ids/0 with addresses: PLAINTEXT://anshul:9092, czxid (broker epoch): 267 (kafka.zk.KafkaZkClient)
[2024-07-09 06:51:56,166] WARN Attempting to send response via channel for which there is no open connection, connection id fe80:0:0:0:447d:591f:badf:5ff5%57:9092-fe80:0:0:0:447d:591f:badf:5ff5%57:62883-6 (kafka.network.Processor)
[2024-07-09 06:51:56,166] INFO [GroupCoordinator 0]: Member kafka-python-2.0.2-29ffda6e-4354-4d04-9805-bd6dca27030f in group mygroup has failed, removing it from the group (kafka.coordinator.group.GroupCoordinator)
[2024-07-09 06:51:56,168] INFO [GroupCoordinator 0]: Preparing to rebalance group mygroup in state PreparingRebalance with old generation 30 (__consumer_offsets-39) (reason: removing member kafka-python-2.0.2-29ffda6e-4354-4d04-9805-bd6dca27030f on heartbeat expiration) (kafka.coordinator.group.GroupCoordinator)
[2024-07-09 06:51:56,169] INFO [GroupCoordinator 0]: Group mygroup with generation 31 is now empty (__consumer_offsets-39) (kafka.coordinator.group.GroupCoordinator)
[2024-07-09 06:51:56,190] INFO [GroupCoordinator 0]: Dynamic Member with unknown member id joins group mygroup in Empty state. Created a new member id kafka-python-2.0.2-43f85858-8694-4b5d-a638-63dc00d394b5 for this member and add to the group. (kafka.coordinator.group.GroupCoordinator)
[2024-07-09 06:51:56,191] INFO [GroupCoordinator 0]: Preparing to rebalance group mygroup in state PreparingRebalance with old generation 31 (__consumer_offsets-39) (reason: Adding new member kafka-python-2.0.2-43f85858-8694-4b5d-a638-63dc00d394b5 with group instance id None; client reason: not provided) (kafka.coordinator.group.GroupCoordinator)
[2024-07-09 06:51:56,194] INFO [GroupCoordinator 0]: Stabilized group mygroup generation 32 (__consumer_offsets-39) with 1 members (kafka.coordinator.group.GroupCoordinator)
[2024-07-09 06:51:56,195] INFO [GroupCoordinator 0]: Assignment received from leader kafka-python-2.0.2-43f85858-8694-4b5d-a638-63dc00d394b5 for group mygroup for generation 32. The group has 1 members, 0 of which are static. (kafka.coordinator.group.GroupCoordinator)
[2024-07-09 06:51:56,342] INFO Expiring session 0x1003acc34310006, timeout of 18000ms exceeded (org.apache.zookeeper.server.ZooKeeperServer)
[2024-07-09 06:51:57,516] WARN fsync-ing the write ahead log in SyncThread:0 took 1091ms which will adversely effect operation latency.File size is 67108880 bytes. See the ZooKeeper troubleshooting guide (org.apache.zookeeper.server.persistence.FileTxnLog)
[2024-07-09 06:51:57,518] WARN Client session timed out, have not heard from server in 4809518ms for session id 0x1003acc34310006 (org.apache.zookeeper.ClientCnxn)
[2024-07-09 06:51:57,520] WARN Session 0x1003acc34310006 for server localhost/[0:0:0:0:0:0:0:1]:2181, Closing socket connection. Attempting reconnect except it is a SessionExpiredException. (org.apache.zookeeper.ClientCnxn)
org.apache.zookeeper.ClientCnxn$SessionTimeoutException: Client session timed out, have not heard from server in 4809518ms for session id 0x1003acc34310006
	at org.apache.zookeeper.ClientCnxn$SendThread.run(ClientCnxn.java:1257)
[2024-07-09 06:51:59,289] INFO Opening socket connection to server localhost/127.0.0.1:2181. (org.apache.zookeeper.ClientCnxn)
[2024-07-09 06:51:59,290] INFO Socket connection established, initiating session, client: /127.0.0.1:62890, server: localhost/127.0.0.1:2181 (org.apache.zookeeper.ClientCnxn)
[2024-07-09 06:51:59,294] INFO Invalid session 0x1003acc34310006 for client /127.0.0.1:62890, probably expired (org.apache.zookeeper.server.ZooKeeperServer)
[2024-07-09 06:51:59,296] WARN Unable to reconnect to ZooKeeper service, session 0x1003acc34310006 has expired (org.apache.zookeeper.ClientCnxn)
[2024-07-09 06:51:59,297] INFO EventThread shut down for session: 0x1003acc34310006 (org.apache.zookeeper.ClientCnxn)
[2024-07-09 06:51:59,297] INFO [ZooKeeperClient Kafka server] Session expired. (kafka.zookeeper.ZooKeeperClient)
[2024-07-09 06:51:59,297] WARN Session 0x1003acc34310006 for server localhost/127.0.0.1:2181, Closing socket connection. Attempting reconnect except it is a SessionExpiredException. (org.apache.zookeeper.ClientCnxn)
org.apache.zookeeper.ClientCnxn$SessionExpiredException: Unable to reconnect to ZooKeeper service, session 0x1003acc34310006 has expired
	at org.apache.zookeeper.ClientCnxn$SendThread.onConnected(ClientCnxn.java:1439)
	at org.apache.zookeeper.ClientCnxnSocket.readConnectResult(ClientCnxnSocket.java:154)
	at org.apache.zookeeper.ClientCnxnSocketNIO.doIO(ClientCnxnSocketNIO.java:86)
	at org.apache.zookeeper.ClientCnxnSocketNIO.doTransport(ClientCnxnSocketNIO.java:350)
	at org.apache.zookeeper.ClientCnxn$SendThread.run(ClientCnxn.java:1289)
[2024-07-09 06:51:59,298] INFO [ZooKeeperClient Kafka server] Initializing a new session to localhost:2181. (kafka.zookeeper.ZooKeeperClient)
[2024-07-09 06:51:59,301] INFO Initiating client connection, connectString=localhost:2181 sessionTimeout=18000 watcher=kafka.zookeeper.ZooKeeperClient$ZooKeeperClientWatcher$@147a5d08 (org.apache.zookeeper.ZooKeeper)
[2024-07-09 06:51:59,305] INFO jute.maxbuffer value is 4194304 Bytes (org.apache.zookeeper.ClientCnxnSocket)
[2024-07-09 06:51:59,306] INFO zookeeper.request.timeout value is 0. feature enabled=false (org.apache.zookeeper.ClientCnxn)
[2024-07-09 06:51:59,308] INFO Opening socket connection to server localhost/127.0.0.1:2181. (org.apache.zookeeper.ClientCnxn)
[2024-07-09 06:51:59,309] INFO Creating /brokers/ids/0 (is it secure? false) (kafka.zk.KafkaZkClient)
[2024-07-09 06:51:59,310] INFO Socket connection established, initiating session, client: /127.0.0.1:62891, server: localhost/127.0.0.1:2181 (org.apache.zookeeper.ClientCnxn)
[2024-07-09 06:51:59,321] INFO Session establishment complete on server localhost/127.0.0.1:2181, session id = 0x1003acc34310007, negotiated timeout = 18000 (org.apache.zookeeper.ClientCnxn)
[2024-07-09 06:51:59,325] INFO [MetadataCache brokerId=0] Updated cache from existing Some(Features(version=3.7-IV4, finalizedFeatures={}, finalizedFeaturesEpoch=0)) to latest Features(version=3.7-IV4, finalizedFeatures={}, finalizedFeaturesEpoch=0). (kafka.server.metadata.ZkMetadataCache)
[2024-07-09 06:51:59,331] INFO Stat of the created znode at /brokers/ids/0 is: 272,272,1720488119324,1720488119324,1,0,0,72122242761293831,196,0,272
 (kafka.zk.KafkaZkClient)
[2024-07-09 06:51:59,334] INFO Registered broker 0 at path /brokers/ids/0 with addresses: PLAINTEXT://anshul:9092, czxid (broker epoch): 272 (kafka.zk.KafkaZkClient)
[2024-07-09 10:49:25,700] INFO [GroupCoordinator 0]: Member kafka-python-2.0.2-43f85858-8694-4b5d-a638-63dc00d394b5 in group mygroup has failed, removing it from the group (kafka.coordinator.group.GroupCoordinator)
[2024-07-09 10:49:25,869] INFO [GroupCoordinator 0]: Preparing to rebalance group mygroup in state PreparingRebalance with old generation 32 (__consumer_offsets-39) (reason: removing member kafka-python-2.0.2-43f85858-8694-4b5d-a638-63dc00d394b5 on heartbeat expiration) (kafka.coordinator.group.GroupCoordinator)
[2024-07-09 10:49:25,870] INFO [GroupCoordinator 0]: Group mygroup with generation 33 is now empty (__consumer_offsets-39) (kafka.coordinator.group.GroupCoordinator)
[2024-07-09 10:49:26,636] INFO [GroupCoordinator 0]: Dynamic Member with unknown member id joins group mygroup in Empty state. Created a new member id kafka-python-2.0.2-f32f7170-1356-4112-ab80-a387723ba0c6 for this member and add to the group. (kafka.coordinator.group.GroupCoordinator)
[2024-07-09 10:49:26,637] INFO [GroupCoordinator 0]: Preparing to rebalance group mygroup in state PreparingRebalance with old generation 33 (__consumer_offsets-39) (reason: Adding new member kafka-python-2.0.2-f32f7170-1356-4112-ab80-a387723ba0c6 with group instance id None; client reason: not provided) (kafka.coordinator.group.GroupCoordinator)
[2024-07-09 10:49:26,638] INFO [GroupCoordinator 0]: Stabilized group mygroup generation 34 (__consumer_offsets-39) with 1 members (kafka.coordinator.group.GroupCoordinator)
[2024-07-09 10:49:26,641] INFO [GroupCoordinator 0]: Assignment received from leader kafka-python-2.0.2-f32f7170-1356-4112-ab80-a387723ba0c6 for group mygroup for generation 34. The group has 1 members, 0 of which are static. (kafka.coordinator.group.GroupCoordinator)
[2024-07-09 10:49:27,086] INFO Expiring session 0x1003acc34310007, timeout of 18000ms exceeded (org.apache.zookeeper.server.ZooKeeperServer)
[2024-07-09 10:49:27,095] WARN Client session timed out, have not heard from server in 14249268ms for session id 0x1003acc34310007 (org.apache.zookeeper.ClientCnxn)
[2024-07-09 10:49:27,095] WARN Session 0x1003acc34310007 for server localhost/127.0.0.1:2181, Closing socket connection. Attempting reconnect except it is a SessionExpiredException. (org.apache.zookeeper.ClientCnxn)
org.apache.zookeeper.ClientCnxn$SessionTimeoutException: Client session timed out, have not heard from server in 14249268ms for session id 0x1003acc34310007
	at org.apache.zookeeper.ClientCnxn$SendThread.run(ClientCnxn.java:1257)
[2024-07-09 10:49:28,972] INFO Opening socket connection to server localhost/127.0.0.1:2181. (org.apache.zookeeper.ClientCnxn)
[2024-07-09 10:49:28,974] INFO Socket connection established, initiating session, client: /127.0.0.1:62896, server: localhost/127.0.0.1:2181 (org.apache.zookeeper.ClientCnxn)
[2024-07-09 10:49:28,977] INFO Invalid session 0x1003acc34310007 for client /127.0.0.1:62896, probably expired (org.apache.zookeeper.server.ZooKeeperServer)
[2024-07-09 10:49:28,980] WARN Unable to reconnect to ZooKeeper service, session 0x1003acc34310007 has expired (org.apache.zookeeper.ClientCnxn)
[2024-07-09 10:49:28,980] INFO EventThread shut down for session: 0x1003acc34310007 (org.apache.zookeeper.ClientCnxn)
[2024-07-09 10:49:28,980] INFO [ZooKeeperClient Kafka server] Session expired. (kafka.zookeeper.ZooKeeperClient)
[2024-07-09 10:49:28,981] WARN Session 0x1003acc34310007 for server localhost/127.0.0.1:2181, Closing socket connection. Attempting reconnect except it is a SessionExpiredException. (org.apache.zookeeper.ClientCnxn)
org.apache.zookeeper.ClientCnxn$SessionExpiredException: Unable to reconnect to ZooKeeper service, session 0x1003acc34310007 has expired
	at org.apache.zookeeper.ClientCnxn$SendThread.onConnected(ClientCnxn.java:1439)
	at org.apache.zookeeper.ClientCnxnSocket.readConnectResult(ClientCnxnSocket.java:154)
	at org.apache.zookeeper.ClientCnxnSocketNIO.doIO(ClientCnxnSocketNIO.java:86)
	at org.apache.zookeeper.ClientCnxnSocketNIO.doTransport(ClientCnxnSocketNIO.java:350)
	at org.apache.zookeeper.ClientCnxn$SendThread.run(ClientCnxn.java:1289)
[2024-07-09 10:49:28,981] INFO [ZooKeeperClient Kafka server] Initializing a new session to localhost:2181. (kafka.zookeeper.ZooKeeperClient)
[2024-07-09 10:49:28,983] INFO Initiating client connection, connectString=localhost:2181 sessionTimeout=18000 watcher=kafka.zookeeper.ZooKeeperClient$ZooKeeperClientWatcher$@147a5d08 (org.apache.zookeeper.ZooKeeper)
[2024-07-09 10:49:28,987] INFO jute.maxbuffer value is 4194304 Bytes (org.apache.zookeeper.ClientCnxnSocket)
[2024-07-09 10:49:28,988] INFO zookeeper.request.timeout value is 0. feature enabled=false (org.apache.zookeeper.ClientCnxn)
[2024-07-09 10:49:28,989] INFO Creating /brokers/ids/0 (is it secure? false) (kafka.zk.KafkaZkClient)
[2024-07-09 10:49:28,990] INFO Opening socket connection to server localhost/[0:0:0:0:0:0:0:1]:2181. (org.apache.zookeeper.ClientCnxn)
[2024-07-09 10:49:28,991] INFO Socket connection established, initiating session, client: /[0:0:0:0:0:0:0:1]:62897, server: localhost/[0:0:0:0:0:0:0:1]:2181 (org.apache.zookeeper.ClientCnxn)
[2024-07-09 10:49:28,997] INFO Session establishment complete on server localhost/[0:0:0:0:0:0:0:1]:2181, session id = 0x1003acc34310008, negotiated timeout = 18000 (org.apache.zookeeper.ClientCnxn)
[2024-07-09 10:49:29,000] INFO [MetadataCache brokerId=0] Updated cache from existing Some(Features(version=3.7-IV4, finalizedFeatures={}, finalizedFeaturesEpoch=0)) to latest Features(version=3.7-IV4, finalizedFeatures={}, finalizedFeaturesEpoch=0). (kafka.server.metadata.ZkMetadataCache)
[2024-07-09 10:49:29,002] INFO Stat of the created znode at /brokers/ids/0 is: 277,277,1720502368999,1720502368999,1,0,0,72122242761293832,196,0,277
 (kafka.zk.KafkaZkClient)
[2024-07-09 10:49:29,003] INFO Registered broker 0 at path /brokers/ids/0 with addresses: PLAINTEXT://anshul:9092, czxid (broker epoch): 277 (kafka.zk.KafkaZkClient)
[2024-07-09 11:09:35,544] INFO Expiring session 0x1003acc34310008, timeout of 18000ms exceeded (org.apache.zookeeper.server.ZooKeeperServer)
[2024-07-09 11:09:35,572] WARN Client session timed out, have not heard from server in 222317ms for session id 0x1003acc34310008 (org.apache.zookeeper.ClientCnxn)
[2024-07-09 11:09:35,576] WARN Session 0x1003acc34310008 for server localhost/[0:0:0:0:0:0:0:1]:2181, Closing socket connection. Attempting reconnect except it is a SessionExpiredException. (org.apache.zookeeper.ClientCnxn)
org.apache.zookeeper.ClientCnxn$SessionTimeoutException: Client session timed out, have not heard from server in 222317ms for session id 0x1003acc34310008
	at org.apache.zookeeper.ClientCnxn$SendThread.run(ClientCnxn.java:1257)
[2024-07-09 11:09:36,843] INFO Opening socket connection to server localhost/127.0.0.1:2181. (org.apache.zookeeper.ClientCnxn)
[2024-07-09 11:09:36,848] INFO Socket connection established, initiating session, client: /127.0.0.1:63927, server: localhost/127.0.0.1:2181 (org.apache.zookeeper.ClientCnxn)
[2024-07-09 11:09:36,852] INFO Invalid session 0x1003acc34310008 for client /127.0.0.1:63927, probably expired (org.apache.zookeeper.server.ZooKeeperServer)
[2024-07-09 11:09:36,854] WARN Unable to reconnect to ZooKeeper service, session 0x1003acc34310008 has expired (org.apache.zookeeper.ClientCnxn)
[2024-07-09 11:09:36,854] INFO EventThread shut down for session: 0x1003acc34310008 (org.apache.zookeeper.ClientCnxn)
[2024-07-09 11:09:36,855] WARN Session 0x1003acc34310008 for server localhost/127.0.0.1:2181, Closing socket connection. Attempting reconnect except it is a SessionExpiredException. (org.apache.zookeeper.ClientCnxn)
org.apache.zookeeper.ClientCnxn$SessionExpiredException: Unable to reconnect to ZooKeeper service, session 0x1003acc34310008 has expired
	at org.apache.zookeeper.ClientCnxn$SendThread.onConnected(ClientCnxn.java:1439)
	at org.apache.zookeeper.ClientCnxnSocket.readConnectResult(ClientCnxnSocket.java:154)
	at org.apache.zookeeper.ClientCnxnSocketNIO.doIO(ClientCnxnSocketNIO.java:86)
	at org.apache.zookeeper.ClientCnxnSocketNIO.doTransport(ClientCnxnSocketNIO.java:350)
	at org.apache.zookeeper.ClientCnxn$SendThread.run(ClientCnxn.java:1289)
[2024-07-09 11:09:36,855] INFO [ZooKeeperClient Kafka server] Session expired. (kafka.zookeeper.ZooKeeperClient)
[2024-07-09 11:09:36,863] INFO [ZooKeeperClient Kafka server] Initializing a new session to localhost:2181. (kafka.zookeeper.ZooKeeperClient)
[2024-07-09 11:09:36,865] INFO Initiating client connection, connectString=localhost:2181 sessionTimeout=18000 watcher=kafka.zookeeper.ZooKeeperClient$ZooKeeperClientWatcher$@147a5d08 (org.apache.zookeeper.ZooKeeper)
[2024-07-09 11:09:36,875] INFO jute.maxbuffer value is 4194304 Bytes (org.apache.zookeeper.ClientCnxnSocket)
[2024-07-09 11:09:36,876] INFO zookeeper.request.timeout value is 0. feature enabled=false (org.apache.zookeeper.ClientCnxn)
[2024-07-09 11:09:36,880] INFO Opening socket connection to server localhost/[0:0:0:0:0:0:0:1]:2181. (org.apache.zookeeper.ClientCnxn)
[2024-07-09 11:09:36,882] INFO Socket connection established, initiating session, client: /[0:0:0:0:0:0:0:1]:63928, server: localhost/[0:0:0:0:0:0:0:1]:2181 (org.apache.zookeeper.ClientCnxn)
[2024-07-09 11:09:36,903] INFO Session establishment complete on server localhost/[0:0:0:0:0:0:0:1]:2181, session id = 0x1003acc34310009, negotiated timeout = 18000 (org.apache.zookeeper.ClientCnxn)
[2024-07-09 11:09:36,930] INFO Creating /brokers/ids/0 (is it secure? false) (kafka.zk.KafkaZkClient)
[2024-07-09 11:09:36,936] INFO [MetadataCache brokerId=0] Updated cache from existing Some(Features(version=3.7-IV4, finalizedFeatures={}, finalizedFeaturesEpoch=0)) to latest Features(version=3.7-IV4, finalizedFeatures={}, finalizedFeaturesEpoch=0). (kafka.server.metadata.ZkMetadataCache)
[2024-07-09 11:09:36,938] INFO Stat of the created znode at /brokers/ids/0 is: 282,282,1720503576935,1720503576935,1,0,0,72122242761293833,196,0,282
 (kafka.zk.KafkaZkClient)
[2024-07-09 11:09:36,939] INFO Registered broker 0 at path /brokers/ids/0 with addresses: PLAINTEXT://anshul:9092, czxid (broker epoch): 282 (kafka.zk.KafkaZkClient)
[2024-07-09 11:43:00,809] INFO Expiring session 0x1003acc34310009, timeout of 18000ms exceeded (org.apache.zookeeper.server.ZooKeeperServer)
[2024-07-09 11:43:00,815] WARN Client session timed out, have not heard from server in 537898ms for session id 0x1003acc34310009 (org.apache.zookeeper.ClientCnxn)
[2024-07-09 11:43:00,816] WARN Session 0x1003acc34310009 for server localhost/[0:0:0:0:0:0:0:1]:2181, Closing socket connection. Attempting reconnect except it is a SessionExpiredException. (org.apache.zookeeper.ClientCnxn)
org.apache.zookeeper.ClientCnxn$SessionTimeoutException: Client session timed out, have not heard from server in 537898ms for session id 0x1003acc34310009
	at org.apache.zookeeper.ClientCnxn$SendThread.run(ClientCnxn.java:1257)
[2024-07-09 11:43:02,477] INFO Opening socket connection to server localhost/[0:0:0:0:0:0:0:1]:2181. (org.apache.zookeeper.ClientCnxn)
[2024-07-09 11:43:02,478] INFO Socket connection established, initiating session, client: /[0:0:0:0:0:0:0:1]:64194, server: localhost/[0:0:0:0:0:0:0:1]:2181 (org.apache.zookeeper.ClientCnxn)
[2024-07-09 11:43:02,480] INFO Invalid session 0x1003acc34310009 for client /[0:0:0:0:0:0:0:1]:64194, probably expired (org.apache.zookeeper.server.ZooKeeperServer)
[2024-07-09 11:43:02,481] INFO EventThread shut down for session: 0x1003acc34310009 (org.apache.zookeeper.ClientCnxn)
[2024-07-09 11:43:02,482] INFO [ZooKeeperClient Kafka server] Session expired. (kafka.zookeeper.ZooKeeperClient)
[2024-07-09 11:43:02,482] WARN Unable to reconnect to ZooKeeper service, session 0x1003acc34310009 has expired (org.apache.zookeeper.ClientCnxn)
[2024-07-09 11:43:02,483] INFO [ZooKeeperClient Kafka server] Initializing a new session to localhost:2181. (kafka.zookeeper.ZooKeeperClient)
[2024-07-09 11:43:02,483] WARN Session 0x1003acc34310009 for server localhost/[0:0:0:0:0:0:0:1]:2181, Closing socket connection. Attempting reconnect except it is a SessionExpiredException. (org.apache.zookeeper.ClientCnxn)
org.apache.zookeeper.ClientCnxn$SessionExpiredException: Unable to reconnect to ZooKeeper service, session 0x1003acc34310009 has expired
	at org.apache.zookeeper.ClientCnxn$SendThread.onConnected(ClientCnxn.java:1439)
	at org.apache.zookeeper.ClientCnxnSocket.readConnectResult(ClientCnxnSocket.java:154)
	at org.apache.zookeeper.ClientCnxnSocketNIO.doIO(ClientCnxnSocketNIO.java:86)
	at org.apache.zookeeper.ClientCnxnSocketNIO.doTransport(ClientCnxnSocketNIO.java:350)
	at org.apache.zookeeper.ClientCnxn$SendThread.run(ClientCnxn.java:1289)
[2024-07-09 11:43:02,483] INFO Initiating client connection, connectString=localhost:2181 sessionTimeout=18000 watcher=kafka.zookeeper.ZooKeeperClient$ZooKeeperClientWatcher$@147a5d08 (org.apache.zookeeper.ZooKeeper)
[2024-07-09 11:43:02,489] INFO jute.maxbuffer value is 4194304 Bytes (org.apache.zookeeper.ClientCnxnSocket)
[2024-07-09 11:43:02,490] INFO zookeeper.request.timeout value is 0. feature enabled=false (org.apache.zookeeper.ClientCnxn)
[2024-07-09 11:43:02,492] INFO Opening socket connection to server localhost/[0:0:0:0:0:0:0:1]:2181. (org.apache.zookeeper.ClientCnxn)
[2024-07-09 11:43:02,492] INFO Creating /brokers/ids/0 (is it secure? false) (kafka.zk.KafkaZkClient)
[2024-07-09 11:43:02,493] INFO Socket connection established, initiating session, client: /[0:0:0:0:0:0:0:1]:64195, server: localhost/[0:0:0:0:0:0:0:1]:2181 (org.apache.zookeeper.ClientCnxn)
[2024-07-09 11:43:02,497] INFO Session establishment complete on server localhost/[0:0:0:0:0:0:0:1]:2181, session id = 0x1003acc3431000a, negotiated timeout = 18000 (org.apache.zookeeper.ClientCnxn)
[2024-07-09 11:43:02,502] INFO [MetadataCache brokerId=0] Updated cache from existing Some(Features(version=3.7-IV4, finalizedFeatures={}, finalizedFeaturesEpoch=0)) to latest Features(version=3.7-IV4, finalizedFeatures={}, finalizedFeaturesEpoch=0). (kafka.server.metadata.ZkMetadataCache)
[2024-07-09 11:43:02,504] INFO Stat of the created znode at /brokers/ids/0 is: 287,287,1720505582500,1720505582500,1,0,0,72122242761293834,196,0,287
 (kafka.zk.KafkaZkClient)
[2024-07-09 11:43:02,505] INFO Registered broker 0 at path /brokers/ids/0 with addresses: PLAINTEXT://anshul:9092, czxid (broker epoch): 287 (kafka.zk.KafkaZkClient)
[2024-07-09 14:46:04,987] INFO [GroupCoordinator 0]: Member kafka-python-2.0.2-f32f7170-1356-4112-ab80-a387723ba0c6 in group mygroup has failed, removing it from the group (kafka.coordinator.group.GroupCoordinator)
[2024-07-09 14:46:04,996] INFO [GroupCoordinator 0]: Preparing to rebalance group mygroup in state PreparingRebalance with old generation 34 (__consumer_offsets-39) (reason: removing member kafka-python-2.0.2-f32f7170-1356-4112-ab80-a387723ba0c6 on heartbeat expiration) (kafka.coordinator.group.GroupCoordinator)
[2024-07-09 14:46:04,998] INFO [GroupCoordinator 0]: Group mygroup with generation 35 is now empty (__consumer_offsets-39) (kafka.coordinator.group.GroupCoordinator)
[2024-07-09 14:46:05,718] INFO [GroupCoordinator 0]: Dynamic Member with unknown member id joins group mygroup in Empty state. Created a new member id kafka-python-2.0.2-831936ae-ee72-472d-9971-9eb7ddf87182 for this member and add to the group. (kafka.coordinator.group.GroupCoordinator)
[2024-07-09 14:46:05,762] INFO [GroupCoordinator 0]: Preparing to rebalance group mygroup in state PreparingRebalance with old generation 35 (__consumer_offsets-39) (reason: Adding new member kafka-python-2.0.2-831936ae-ee72-472d-9971-9eb7ddf87182 with group instance id None; client reason: not provided) (kafka.coordinator.group.GroupCoordinator)
[2024-07-09 14:46:05,810] INFO [GroupCoordinator 0]: Stabilized group mygroup generation 36 (__consumer_offsets-39) with 1 members (kafka.coordinator.group.GroupCoordinator)
[2024-07-09 14:46:05,854] INFO [GroupCoordinator 0]: Assignment received from leader kafka-python-2.0.2-831936ae-ee72-472d-9971-9eb7ddf87182 for group mygroup for generation 36. The group has 1 members, 0 of which are static. (kafka.coordinator.group.GroupCoordinator)
[2024-07-09 14:46:06,898] WARN Client session timed out, have not heard from server in 1058544ms for session id 0x1003acc3431000a (org.apache.zookeeper.ClientCnxn)
[2024-07-09 14:46:06,900] WARN Session 0x1003acc3431000a for server localhost/[0:0:0:0:0:0:0:1]:2181, Closing socket connection. Attempting reconnect except it is a SessionExpiredException. (org.apache.zookeeper.ClientCnxn)
org.apache.zookeeper.ClientCnxn$SessionTimeoutException: Client session timed out, have not heard from server in 1058544ms for session id 0x1003acc3431000a
	at org.apache.zookeeper.ClientCnxn$SendThread.run(ClientCnxn.java:1257)
[2024-07-09 14:46:06,916] INFO Unable to read additional data from client, it probably closed the socket: address = /[0:0:0:0:0:0:0:1]:64195, session = 0x1003acc3431000a (org.apache.zookeeper.server.NIOServerCnxn)
[2024-07-09 14:46:07,511] INFO Expiring session 0x1003acc3431000a, timeout of 18000ms exceeded (org.apache.zookeeper.server.ZooKeeperServer)
[2024-07-09 14:46:08,717] INFO Opening socket connection to server localhost/127.0.0.1:2181. (org.apache.zookeeper.ClientCnxn)
[2024-07-09 14:46:08,720] INFO Socket connection established, initiating session, client: /127.0.0.1:51829, server: localhost/127.0.0.1:2181 (org.apache.zookeeper.ClientCnxn)
[2024-07-09 14:46:08,725] INFO Invalid session 0x1003acc3431000a for client /127.0.0.1:51829, probably expired (org.apache.zookeeper.server.ZooKeeperServer)
[2024-07-09 14:46:08,727] WARN Unable to reconnect to ZooKeeper service, session 0x1003acc3431000a has expired (org.apache.zookeeper.ClientCnxn)
[2024-07-09 14:46:08,728] INFO EventThread shut down for session: 0x1003acc3431000a (org.apache.zookeeper.ClientCnxn)
[2024-07-09 14:46:08,728] INFO [ZooKeeperClient Kafka server] Session expired. (kafka.zookeeper.ZooKeeperClient)
[2024-07-09 14:46:08,728] WARN Session 0x1003acc3431000a for server localhost/127.0.0.1:2181, Closing socket connection. Attempting reconnect except it is a SessionExpiredException. (org.apache.zookeeper.ClientCnxn)
org.apache.zookeeper.ClientCnxn$SessionExpiredException: Unable to reconnect to ZooKeeper service, session 0x1003acc3431000a has expired
	at org.apache.zookeeper.ClientCnxn$SendThread.onConnected(ClientCnxn.java:1439)
	at org.apache.zookeeper.ClientCnxnSocket.readConnectResult(ClientCnxnSocket.java:154)
	at org.apache.zookeeper.ClientCnxnSocketNIO.doIO(ClientCnxnSocketNIO.java:86)
	at org.apache.zookeeper.ClientCnxnSocketNIO.doTransport(ClientCnxnSocketNIO.java:350)
	at org.apache.zookeeper.ClientCnxn$SendThread.run(ClientCnxn.java:1289)
[2024-07-09 14:46:08,732] INFO [ZooKeeperClient Kafka server] Initializing a new session to localhost:2181. (kafka.zookeeper.ZooKeeperClient)
[2024-07-09 14:46:08,733] INFO Initiating client connection, connectString=localhost:2181 sessionTimeout=18000 watcher=kafka.zookeeper.ZooKeeperClient$ZooKeeperClientWatcher$@147a5d08 (org.apache.zookeeper.ZooKeeper)
[2024-07-09 14:46:08,743] INFO jute.maxbuffer value is 4194304 Bytes (org.apache.zookeeper.ClientCnxnSocket)
[2024-07-09 14:46:08,744] INFO zookeeper.request.timeout value is 0. feature enabled=false (org.apache.zookeeper.ClientCnxn)
[2024-07-09 14:46:08,746] INFO Opening socket connection to server localhost/127.0.0.1:2181. (org.apache.zookeeper.ClientCnxn)
[2024-07-09 14:46:08,747] INFO Socket connection established, initiating session, client: /127.0.0.1:51830, server: localhost/127.0.0.1:2181 (org.apache.zookeeper.ClientCnxn)
[2024-07-09 14:46:08,754] INFO Session establishment complete on server localhost/127.0.0.1:2181, session id = 0x1003acc3431000b, negotiated timeout = 18000 (org.apache.zookeeper.ClientCnxn)
[2024-07-09 14:46:08,760] INFO Creating /brokers/ids/0 (is it secure? false) (kafka.zk.KafkaZkClient)
[2024-07-09 14:46:08,763] INFO [MetadataCache brokerId=0] Updated cache from existing Some(Features(version=3.7-IV4, finalizedFeatures={}, finalizedFeaturesEpoch=0)) to latest Features(version=3.7-IV4, finalizedFeatures={}, finalizedFeaturesEpoch=0). (kafka.server.metadata.ZkMetadataCache)
[2024-07-09 14:46:08,767] INFO Stat of the created znode at /brokers/ids/0 is: 292,292,1720516568763,1720516568763,1,0,0,72122242761293835,196,0,292
 (kafka.zk.KafkaZkClient)
[2024-07-09 14:46:08,769] INFO Registered broker 0 at path /brokers/ids/0 with addresses: PLAINTEXT://anshul:9092, czxid (broker epoch): 292 (kafka.zk.KafkaZkClient)
[2024-07-09 15:39:31,838] WARN Attempting to send response via channel for which there is no open connection, connection id fe80:0:0:0:447d:591f:badf:5ff5%57:9092-fe80:0:0:0:447d:591f:badf:5ff5%57:51824-10 (kafka.network.Processor)
[2024-07-09 15:39:31,970] INFO [GroupCoordinator 0]: Member kafka-python-2.0.2-831936ae-ee72-472d-9971-9eb7ddf87182 in group mygroup has failed, removing it from the group (kafka.coordinator.group.GroupCoordinator)
[2024-07-09 15:39:31,975] INFO [GroupCoordinator 0]: Preparing to rebalance group mygroup in state PreparingRebalance with old generation 36 (__consumer_offsets-39) (reason: removing member kafka-python-2.0.2-831936ae-ee72-472d-9971-9eb7ddf87182 on heartbeat expiration) (kafka.coordinator.group.GroupCoordinator)
[2024-07-09 15:39:31,978] INFO [GroupCoordinator 0]: Group mygroup with generation 37 is now empty (__consumer_offsets-39) (kafka.coordinator.group.GroupCoordinator)
[2024-07-09 15:39:32,547] INFO [GroupCoordinator 0]: Dynamic Member with unknown member id joins group mygroup in Empty state. Created a new member id kafka-python-2.0.2-0fe728a7-6bf1-4cba-a2f1-acec0940fa15 for this member and add to the group. (kafka.coordinator.group.GroupCoordinator)
[2024-07-09 15:39:32,554] INFO [GroupCoordinator 0]: Preparing to rebalance group mygroup in state PreparingRebalance with old generation 37 (__consumer_offsets-39) (reason: Adding new member kafka-python-2.0.2-0fe728a7-6bf1-4cba-a2f1-acec0940fa15 with group instance id None; client reason: not provided) (kafka.coordinator.group.GroupCoordinator)
[2024-07-09 15:39:32,560] INFO [GroupCoordinator 0]: Stabilized group mygroup generation 38 (__consumer_offsets-39) with 1 members (kafka.coordinator.group.GroupCoordinator)
[2024-07-09 15:39:32,567] INFO [GroupCoordinator 0]: Assignment received from leader kafka-python-2.0.2-0fe728a7-6bf1-4cba-a2f1-acec0940fa15 for group mygroup for generation 38. The group has 1 members, 0 of which are static. (kafka.coordinator.group.GroupCoordinator)
[2024-07-09 15:39:34,147] INFO Expiring session 0x1003acc3431000b, timeout of 18000ms exceeded (org.apache.zookeeper.server.ZooKeeperServer)
[2024-07-09 15:39:34,238] WARN Client session timed out, have not heard from server in 964001ms for session id 0x1003acc3431000b (org.apache.zookeeper.ClientCnxn)
[2024-07-09 15:39:34,244] WARN Session 0x1003acc3431000b for server localhost/127.0.0.1:2181, Closing socket connection. Attempting reconnect except it is a SessionExpiredException. (org.apache.zookeeper.ClientCnxn)
org.apache.zookeeper.ClientCnxn$SessionTimeoutException: Client session timed out, have not heard from server in 964001ms for session id 0x1003acc3431000b
	at org.apache.zookeeper.ClientCnxn$SendThread.run(ClientCnxn.java:1257)
[2024-07-09 15:39:35,615] INFO Opening socket connection to server localhost/127.0.0.1:2181. (org.apache.zookeeper.ClientCnxn)
[2024-07-09 15:39:35,621] INFO Socket connection established, initiating session, client: /127.0.0.1:52176, server: localhost/127.0.0.1:2181 (org.apache.zookeeper.ClientCnxn)
[2024-07-09 15:39:35,629] INFO Invalid session 0x1003acc3431000b for client /127.0.0.1:52176, probably expired (org.apache.zookeeper.server.ZooKeeperServer)
[2024-07-09 15:39:35,632] WARN Unable to reconnect to ZooKeeper service, session 0x1003acc3431000b has expired (org.apache.zookeeper.ClientCnxn)
[2024-07-09 15:39:35,632] INFO EventThread shut down for session: 0x1003acc3431000b (org.apache.zookeeper.ClientCnxn)
[2024-07-09 15:39:35,633] INFO [ZooKeeperClient Kafka server] Session expired. (kafka.zookeeper.ZooKeeperClient)
[2024-07-09 15:39:35,633] WARN Session 0x1003acc3431000b for server localhost/127.0.0.1:2181, Closing socket connection. Attempting reconnect except it is a SessionExpiredException. (org.apache.zookeeper.ClientCnxn)
org.apache.zookeeper.ClientCnxn$SessionExpiredException: Unable to reconnect to ZooKeeper service, session 0x1003acc3431000b has expired
	at org.apache.zookeeper.ClientCnxn$SendThread.onConnected(ClientCnxn.java:1439)
	at org.apache.zookeeper.ClientCnxnSocket.readConnectResult(ClientCnxnSocket.java:154)
	at org.apache.zookeeper.ClientCnxnSocketNIO.doIO(ClientCnxnSocketNIO.java:86)
	at org.apache.zookeeper.ClientCnxnSocketNIO.doTransport(ClientCnxnSocketNIO.java:350)
	at org.apache.zookeeper.ClientCnxn$SendThread.run(ClientCnxn.java:1289)
[2024-07-09 15:39:35,637] INFO [ZooKeeperClient Kafka server] Initializing a new session to localhost:2181. (kafka.zookeeper.ZooKeeperClient)
[2024-07-09 15:39:35,640] INFO Initiating client connection, connectString=localhost:2181 sessionTimeout=18000 watcher=kafka.zookeeper.ZooKeeperClient$ZooKeeperClientWatcher$@147a5d08 (org.apache.zookeeper.ZooKeeper)
[2024-07-09 15:39:35,650] INFO jute.maxbuffer value is 4194304 Bytes (org.apache.zookeeper.ClientCnxnSocket)
[2024-07-09 15:39:35,651] INFO zookeeper.request.timeout value is 0. feature enabled=false (org.apache.zookeeper.ClientCnxn)
[2024-07-09 15:39:35,653] INFO Opening socket connection to server localhost/127.0.0.1:2181. (org.apache.zookeeper.ClientCnxn)
[2024-07-09 15:39:35,654] INFO Socket connection established, initiating session, client: /127.0.0.1:52177, server: localhost/127.0.0.1:2181 (org.apache.zookeeper.ClientCnxn)
[2024-07-09 15:39:35,662] INFO Session establishment complete on server localhost/127.0.0.1:2181, session id = 0x1003acc3431000c, negotiated timeout = 18000 (org.apache.zookeeper.ClientCnxn)
[2024-07-09 15:39:35,667] INFO Creating /brokers/ids/0 (is it secure? false) (kafka.zk.KafkaZkClient)
[2024-07-09 15:39:35,669] INFO [MetadataCache brokerId=0] Updated cache from existing Some(Features(version=3.7-IV4, finalizedFeatures={}, finalizedFeaturesEpoch=0)) to latest Features(version=3.7-IV4, finalizedFeatures={}, finalizedFeaturesEpoch=0). (kafka.server.metadata.ZkMetadataCache)
[2024-07-09 15:39:35,681] INFO Stat of the created znode at /brokers/ids/0 is: 297,297,1720519775677,1720519775677,1,0,0,72122242761293836,196,0,297
 (kafka.zk.KafkaZkClient)
[2024-07-09 15:39:35,682] INFO Registered broker 0 at path /brokers/ids/0 with addresses: PLAINTEXT://anshul:9092, czxid (broker epoch): 297 (kafka.zk.KafkaZkClient)
[2024-07-10 14:49:29,940] INFO [GroupCoordinator 0]: Member kafka-python-2.0.2-0fe728a7-6bf1-4cba-a2f1-acec0940fa15 in group mygroup has failed, removing it from the group (kafka.coordinator.group.GroupCoordinator)
[2024-07-10 14:49:29,958] WARN Attempting to send response via channel for which there is no open connection, connection id 172.27.48.1:9092-172.27.48.1:52153-11 (kafka.network.Processor)
[2024-07-10 14:49:30,810] INFO [GroupCoordinator 0]: Preparing to rebalance group mygroup in state PreparingRebalance with old generation 38 (__consumer_offsets-39) (reason: removing member kafka-python-2.0.2-0fe728a7-6bf1-4cba-a2f1-acec0940fa15 on heartbeat expiration) (kafka.coordinator.group.GroupCoordinator)
[2024-07-10 14:49:30,814] INFO [GroupCoordinator 0]: Group mygroup with generation 39 is now empty (__consumer_offsets-39) (kafka.coordinator.group.GroupCoordinator)
[2024-07-10 14:49:30,343] INFO Expiring session 0x1003acc3431000c, timeout of 18000ms exceeded (org.apache.zookeeper.server.ZooKeeperServer)
[2024-07-10 14:49:30,920] WARN Client session timed out, have not heard from server in 80044397ms for session id 0x1003acc3431000c (org.apache.zookeeper.ClientCnxn)
[2024-07-10 14:49:30,927] WARN Session 0x1003acc3431000c for server localhost/127.0.0.1:2181, Closing socket connection. Attempting reconnect except it is a SessionExpiredException. (org.apache.zookeeper.ClientCnxn)
org.apache.zookeeper.ClientCnxn$SessionTimeoutException: Client session timed out, have not heard from server in 80044397ms for session id 0x1003acc3431000c
	at org.apache.zookeeper.ClientCnxn$SendThread.run(ClientCnxn.java:1257)
[2024-07-10 14:49:31,092] INFO [GroupCoordinator 0]: Dynamic Member with unknown member id joins group mygroup in Empty state. Created a new member id kafka-python-2.0.2-3dbe51d7-0ee0-4216-a45b-dc0331f38ed3 for this member and add to the group. (kafka.coordinator.group.GroupCoordinator)
[2024-07-10 14:49:31,100] INFO [GroupCoordinator 0]: Preparing to rebalance group mygroup in state PreparingRebalance with old generation 39 (__consumer_offsets-39) (reason: Adding new member kafka-python-2.0.2-3dbe51d7-0ee0-4216-a45b-dc0331f38ed3 with group instance id None; client reason: not provided) (kafka.coordinator.group.GroupCoordinator)
[2024-07-10 14:49:31,115] INFO [GroupCoordinator 0]: Stabilized group mygroup generation 40 (__consumer_offsets-39) with 1 members (kafka.coordinator.group.GroupCoordinator)
[2024-07-10 14:49:31,134] INFO [GroupCoordinator 0]: Assignment received from leader kafka-python-2.0.2-3dbe51d7-0ee0-4216-a45b-dc0331f38ed3 for group mygroup for generation 40. The group has 1 members, 0 of which are static. (kafka.coordinator.group.GroupCoordinator)
[2024-07-10 14:49:32,471] INFO Opening socket connection to server localhost/127.0.0.1:2181. (org.apache.zookeeper.ClientCnxn)
[2024-07-10 14:49:32,489] INFO Socket connection established, initiating session, client: /127.0.0.1:52876, server: localhost/127.0.0.1:2181 (org.apache.zookeeper.ClientCnxn)
[2024-07-10 14:49:32,567] INFO Invalid session 0x1003acc3431000c for client /127.0.0.1:52876, probably expired (org.apache.zookeeper.server.ZooKeeperServer)
[2024-07-10 14:49:32,588] WARN Unable to reconnect to ZooKeeper service, session 0x1003acc3431000c has expired (org.apache.zookeeper.ClientCnxn)
[2024-07-10 14:49:32,593] WARN Session 0x1003acc3431000c for server localhost/127.0.0.1:2181, Closing socket connection. Attempting reconnect except it is a SessionExpiredException. (org.apache.zookeeper.ClientCnxn)
org.apache.zookeeper.ClientCnxn$SessionExpiredException: Unable to reconnect to ZooKeeper service, session 0x1003acc3431000c has expired
	at org.apache.zookeeper.ClientCnxn$SendThread.onConnected(ClientCnxn.java:1439)
	at org.apache.zookeeper.ClientCnxnSocket.readConnectResult(ClientCnxnSocket.java:154)
	at org.apache.zookeeper.ClientCnxnSocketNIO.doIO(ClientCnxnSocketNIO.java:86)
	at org.apache.zookeeper.ClientCnxnSocketNIO.doTransport(ClientCnxnSocketNIO.java:350)
	at org.apache.zookeeper.ClientCnxn$SendThread.run(ClientCnxn.java:1289)
[2024-07-10 14:49:32,598] INFO [ZooKeeperClient Kafka server] Session expired. (kafka.zookeeper.ZooKeeperClient)
[2024-07-10 14:49:32,597] INFO EventThread shut down for session: 0x1003acc3431000c (org.apache.zookeeper.ClientCnxn)
[2024-07-10 14:49:32,646] INFO [ZooKeeperClient Kafka server] Initializing a new session to localhost:2181. (kafka.zookeeper.ZooKeeperClient)
[2024-07-10 14:49:32,653] INFO Initiating client connection, connectString=localhost:2181 sessionTimeout=18000 watcher=kafka.zookeeper.ZooKeeperClient$ZooKeeperClientWatcher$@147a5d08 (org.apache.zookeeper.ZooKeeper)
[2024-07-10 14:49:32,692] INFO jute.maxbuffer value is 4194304 Bytes (org.apache.zookeeper.ClientCnxnSocket)
[2024-07-10 14:49:32,695] INFO zookeeper.request.timeout value is 0. feature enabled=false (org.apache.zookeeper.ClientCnxn)
[2024-07-10 14:49:32,698] INFO Opening socket connection to server localhost/[0:0:0:0:0:0:0:1]:2181. (org.apache.zookeeper.ClientCnxn)
[2024-07-10 14:49:32,700] INFO Socket connection established, initiating session, client: /[0:0:0:0:0:0:0:1]:52877, server: localhost/[0:0:0:0:0:0:0:1]:2181 (org.apache.zookeeper.ClientCnxn)
[2024-07-10 14:49:32,731] INFO Session establishment complete on server localhost/[0:0:0:0:0:0:0:1]:2181, session id = 0x1003acc3431000d, negotiated timeout = 18000 (org.apache.zookeeper.ClientCnxn)
[2024-07-10 14:49:32,773] INFO [MetadataCache brokerId=0] Updated cache from existing Some(Features(version=3.7-IV4, finalizedFeatures={}, finalizedFeaturesEpoch=0)) to latest Features(version=3.7-IV4, finalizedFeatures={}, finalizedFeaturesEpoch=0). (kafka.server.metadata.ZkMetadataCache)
[2024-07-10 14:49:32,779] INFO Creating /brokers/ids/0 (is it secure? false) (kafka.zk.KafkaZkClient)
[2024-07-10 14:49:32,803] INFO Stat of the created znode at /brokers/ids/0 is: 302,302,1720603172789,1720603172789,1,0,0,72122242761293837,196,0,302
 (kafka.zk.KafkaZkClient)
[2024-07-10 14:49:32,803] INFO Registered broker 0 at path /brokers/ids/0 with addresses: PLAINTEXT://anshul:9092, czxid (broker epoch): 302 (kafka.zk.KafkaZkClient)
[2024-07-10 15:39:22,683] WARN Attempting to send response via channel for which there is no open connection, connection id fe80:0:0:0:46de:e134:6310:13bc%18:9092-fe80:0:0:0:46de:e134:6310:13bc%18:52875-12 (kafka.network.Processor)
[2024-07-10 15:39:22,715] INFO [GroupCoordinator 0]: Member kafka-python-2.0.2-3dbe51d7-0ee0-4216-a45b-dc0331f38ed3 in group mygroup has failed, removing it from the group (kafka.coordinator.group.GroupCoordinator)
[2024-07-10 15:39:22,730] INFO [GroupCoordinator 0]: Preparing to rebalance group mygroup in state PreparingRebalance with old generation 40 (__consumer_offsets-39) (reason: removing member kafka-python-2.0.2-3dbe51d7-0ee0-4216-a45b-dc0331f38ed3 on heartbeat expiration) (kafka.coordinator.group.GroupCoordinator)
[2024-07-10 15:39:22,746] INFO [GroupCoordinator 0]: Group mygroup with generation 41 is now empty (__consumer_offsets-39) (kafka.coordinator.group.GroupCoordinator)
[2024-07-10 15:39:22,809] INFO [GroupCoordinator 0]: Dynamic Member with unknown member id joins group mygroup in Empty state. Created a new member id kafka-python-2.0.2-97975136-5a5a-466b-a812-66da596dc5f0 for this member and add to the group. (kafka.coordinator.group.GroupCoordinator)
[2024-07-10 15:39:22,817] INFO [GroupCoordinator 0]: Preparing to rebalance group mygroup in state PreparingRebalance with old generation 41 (__consumer_offsets-39) (reason: Adding new member kafka-python-2.0.2-97975136-5a5a-466b-a812-66da596dc5f0 with group instance id None; client reason: not provided) (kafka.coordinator.group.GroupCoordinator)
[2024-07-10 15:39:22,817] INFO [GroupCoordinator 0]: Stabilized group mygroup generation 42 (__consumer_offsets-39) with 1 members (kafka.coordinator.group.GroupCoordinator)
[2024-07-10 15:39:22,833] INFO [GroupCoordinator 0]: Assignment received from leader kafka-python-2.0.2-97975136-5a5a-466b-a812-66da596dc5f0 for group mygroup for generation 42. The group has 1 members, 0 of which are static. (kafka.coordinator.group.GroupCoordinator)
[2024-07-10 15:39:24,430] WARN Session 0x1003acc3431000d for server localhost/[0:0:0:0:0:0:0:1]:2181, Closing socket connection. Attempting reconnect except it is a SessionExpiredException. (org.apache.zookeeper.ClientCnxn)
EndOfStreamException: Unable to read additional data from server sessionid 0x1003acc3431000d, likely server has closed socket
	at org.apache.zookeeper.ClientCnxnSocketNIO.doIO(ClientCnxnSocketNIO.java:77)
	at org.apache.zookeeper.ClientCnxnSocketNIO.doTransport(ClientCnxnSocketNIO.java:350)
	at org.apache.zookeeper.ClientCnxn$SendThread.run(ClientCnxn.java:1289)
[2024-07-10 15:39:25,318] INFO Expiring session 0x1003acc3431000d, timeout of 18000ms exceeded (org.apache.zookeeper.server.ZooKeeperServer)
[2024-07-10 15:39:26,575] INFO Opening socket connection to server localhost/[0:0:0:0:0:0:0:1]:2181. (org.apache.zookeeper.ClientCnxn)
[2024-07-10 15:39:26,591] INFO Socket connection established, initiating session, client: /[0:0:0:0:0:0:0:1]:54008, server: localhost/[0:0:0:0:0:0:0:1]:2181 (org.apache.zookeeper.ClientCnxn)
[2024-07-10 15:39:26,639] INFO Invalid session 0x1003acc3431000d for client /[0:0:0:0:0:0:0:1]:54008, probably expired (org.apache.zookeeper.server.ZooKeeperServer)
[2024-07-10 15:39:26,639] WARN Unable to reconnect to ZooKeeper service, session 0x1003acc3431000d has expired (org.apache.zookeeper.ClientCnxn)
[2024-07-10 15:39:26,639] WARN Session 0x1003acc3431000d for server localhost/[0:0:0:0:0:0:0:1]:2181, Closing socket connection. Attempting reconnect except it is a SessionExpiredException. (org.apache.zookeeper.ClientCnxn)
org.apache.zookeeper.ClientCnxn$SessionExpiredException: Unable to reconnect to ZooKeeper service, session 0x1003acc3431000d has expired
	at org.apache.zookeeper.ClientCnxn$SendThread.onConnected(ClientCnxn.java:1439)
	at org.apache.zookeeper.ClientCnxnSocket.readConnectResult(ClientCnxnSocket.java:154)
	at org.apache.zookeeper.ClientCnxnSocketNIO.doIO(ClientCnxnSocketNIO.java:86)
	at org.apache.zookeeper.ClientCnxnSocketNIO.doTransport(ClientCnxnSocketNIO.java:350)
	at org.apache.zookeeper.ClientCnxn$SendThread.run(ClientCnxn.java:1289)
[2024-07-10 15:39:26,654] INFO [ZooKeeperClient Kafka server] Session expired. (kafka.zookeeper.ZooKeeperClient)
[2024-07-10 15:39:26,639] INFO EventThread shut down for session: 0x1003acc3431000d (org.apache.zookeeper.ClientCnxn)
[2024-07-10 15:39:26,670] INFO [ZooKeeperClient Kafka server] Initializing a new session to localhost:2181. (kafka.zookeeper.ZooKeeperClient)
[2024-07-10 15:39:26,670] INFO Initiating client connection, connectString=localhost:2181 sessionTimeout=18000 watcher=kafka.zookeeper.ZooKeeperClient$ZooKeeperClientWatcher$@147a5d08 (org.apache.zookeeper.ZooKeeper)
[2024-07-10 15:39:26,733] INFO jute.maxbuffer value is 4194304 Bytes (org.apache.zookeeper.ClientCnxnSocket)
[2024-07-10 15:39:26,733] INFO zookeeper.request.timeout value is 0. feature enabled=false (org.apache.zookeeper.ClientCnxn)
[2024-07-10 15:39:26,749] INFO Opening socket connection to server localhost/[0:0:0:0:0:0:0:1]:2181. (org.apache.zookeeper.ClientCnxn)
[2024-07-10 15:39:26,749] INFO Socket connection established, initiating session, client: /[0:0:0:0:0:0:0:1]:54011, server: localhost/[0:0:0:0:0:0:0:1]:2181 (org.apache.zookeeper.ClientCnxn)
[2024-07-10 15:39:26,780] INFO Session establishment complete on server localhost/[0:0:0:0:0:0:0:1]:2181, session id = 0x1003acc3431000e, negotiated timeout = 18000 (org.apache.zookeeper.ClientCnxn)
[2024-07-10 15:39:26,815] INFO Creating /brokers/ids/0 (is it secure? false) (kafka.zk.KafkaZkClient)
[2024-07-10 15:39:26,827] INFO [MetadataCache brokerId=0] Updated cache from existing Some(Features(version=3.7-IV4, finalizedFeatures={}, finalizedFeaturesEpoch=0)) to latest Features(version=3.7-IV4, finalizedFeatures={}, finalizedFeaturesEpoch=0). (kafka.server.metadata.ZkMetadataCache)
[2024-07-10 15:39:27,110] INFO Stat of the created znode at /brokers/ids/0 is: 307,307,1720606166827,1720606166827,1,0,0,72122242761293838,196,0,307
 (kafka.zk.KafkaZkClient)
[2024-07-10 15:39:27,114] INFO Registered broker 0 at path /brokers/ids/0 with addresses: PLAINTEXT://anshul:9092, czxid (broker epoch): 307 (kafka.zk.KafkaZkClient)
